### PR TITLE
Add Duration overload to SlowLoadableComponent constructor

### PR DESCRIPTION
**Pull request by kluever**
### Description
Adds a `Duration`-based overload to `SlowLoadableComponent` constructor.
Also renames a variable, and fixes an import.

### Motivation and Context
Allows users to pass in a full `Duration`, instead of forcing them to decompose it.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by kluever** [comment]
Getting an `Internal Server Error` when I try to sign the CLA...

File: java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java [hunk 1]
```diff
@@ -36,11 +36,15 @@ public abstract class SlowLoadableComponent<T extends LoadableComponent<T>>
     extends LoadableComponent<T> {
 
   private final Clock clock;
-  private final Duration timeOutInSeconds;
+  private final Duration timeOut;
 
-  public SlowLoadableComponent(java.time.Clock clock, int timeOutInSeconds) {
+  public SlowLoadableComponent(Clock clock, int timeOutInSeconds) {
+    this(clock, Duration.ofSeconds(timeOutInSeconds));
+  }
```
Comment (diemol) [inline java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java:47]: Why don't you deprecate this one so we eventually only leave the new one?

Comment (diemol) [reply java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java:47]: And perhaps add a JavaDoc to tell the user.

**Review by diemol — CHANGES_REQUESTED** [review]
We are not usually accepting changes on support packages because we want to move them to a different project. However, we are moving to use `Duration` wherever we can in Java, and this change is minor. 

Can you please check the comments so we can merge this?

File: java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java (hunk 1 above)
Comment (kluever) [reply java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java:47]: Sure, done!

Comment (diemol) [reply java/src/org/openqa/selenium/support/ui/SlowLoadableComponent.java:47]: Thank you, @kluever!

**Review by diemol — COMMENTED** [review]
Can you please sign the CLA?

**Comment by kluever** [comment]
CLA is done --- was having trouble w/ the 3p CLA website last week.

**Review by diemol — APPROVED** [review]
Thank you, @kluever!

**Pull request merged**